packages/react/src/components/Select/Select.tsx

```diff
@@ -255,6 +272,10 @@ const Select = React.forwardRef(function Select(
       size: 'mini',
     });
   }
+  useEffect(() => {
+    selectDefaultTitle(defaultValue);
```

Comment:
This doesn't need to be in a function, just place the code directly in the useEffect. It'll be easier to maintain this way being all in once place and not having to scroll up to find the function definition.

Reply:
Done